### DISCUSSION TITLE
Luau benchmarks to compare against Lua and LuaJIT

### DISCUSSION THREAD

**Discussion by bandaloo** [hanabi1224/Programming-Language-Benchmarks#364] [Ideas]
I'd be interested in contributing this if I have the time!

Luau is a language inspired by Lua being worked on by Roblox. It has gradual typing and claims to ship with an optimized version of the Lua standard libraries, but is not JIT'ed. Very easy to build from source but they also ship standalone binaries. Since Roblox has a very specific use case, I'm sure the optimizations they make are not general purpose and mostly oriented around games, so it would be cool to see where (and if) their custom runtime happens to outperform the JIT.

Not sure if the language is too niche, however.

**Comment by hanabi1224** [comment]
Sounds good : )
